src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java

```diff
@@ -21,7 +22,7 @@ public class PermissionsAPIController {
     UserRoleMapService userRoleMapService;
 
     @Autowired
-    UserRoleService userRoleService;
+    UserRoleService hm, userRoleService;
```

Comment:
typo? (hm,)

Reply:
Yes. I totally missed that, and everything still ran as it was supposed to. Corrected and verified tests and Postman.